components/UploadImageModal/UploadImageModal.styled.js

```diff
@@ -0,0 +1,26 @@
+import styled from "styled-components";
+
+export const Header = styled.h2`
+  font-size: 1rem;
+  margin-bottom: 30px;
+`;
+
+export const FileInput = styled.input`
+  &::file-selector-button {
+    background-color: white;
+    border: 1px solid grey;
```

Comment:
Also here I suggest you  to choose hexcodes instead of the color names.

Reply:
grey: #808080